YDotNet/Document/Events/EventSubscriber.cs

```diff
@@ -5,15 +5,17 @@ internal class EventSubscriber<TEvent> : IEventSubscriber
     private readonly EventManager manager;
```

Comment:
Do we still need this file? I mean it is potentially unsafe to use the raw pointer.

Reply:
No, we can't drop it because `Doc` and `UndoManager` aren't subclasses of `Branch` and still need this type of event manager.